tracer/src/Datadog.Trace/Configuration/ConfigurationKeys.Debugger.cs

```diff
@@ -143,7 +143,7 @@ internal static class Debugger
             /// Configuration key for the maximum number of frames in a call stack we would like to capture values for.
             /// </summary>
             /// <seealso cref="ExceptionReplaySettings.MaximumFramesToCapture"/>
-            public const string ExceptionReplayMaxFramesToCapture = "DD_EXCEPTION_REPLAY_MAX_FRAMES_TO_CAPTURE";
+            public const string ExceptionReplayCaptureMaxFrames = "DD_EXCEPTION_REPLAY_CAPTURE_MAX_FRAMES";
```

Comment:
do we need to update the go repo also?

Reply:
yes, I'll open a separate PR once the CI/CD will be sane again

Reply:
> once the CI/CD will be sane again

FYI, I believe you now need to update and merge that in the dd-go repo _before_ merging this, otherwise System Tests will fail on master😬

Reply:
Unrelated, but IMO, the previous configuration key had a better name 😉 

Also, are any customers currently using this? If so, are we happy making a breaking change here?

Reply:
Nobody uses it (except internally by our System Tests), and yes we currently having this discussion on what the name should be